src/utils/colors/index.js

```diff
@@ -2,35 +2,42 @@
 export const colors = {
   bondi_blue: '#00ADB5',
   white: '#ffffff',
-  black: '#000000',
+  black: '#0C0C0C',
   emperor: '#4F4F4F',
-  gray: '#828282',
-  gray1: '#C4C4C4',
-  lightgrey: '#F5F5F5',
+  gray: '#69707C',
+  gray1: '#9B9FA9',
+  lightgrey: '#FFFFFF',
   concrete: '#F2F2F2',
   mine_shaft: '#333333',
   bunting: '#11243D',
-  alto: '#E0E0E0',
+  alto: '#E8EBED',
   porcelain: '#ecf0f1',
   silver: '#BDBDBD',
   french_pass: '#b6e4fd',
   pattens_blue: '#ddf2fe',
   blue: '#2F80ED',
   lightblue: '#55C2FF',
-  blue1: '#2C67BC',
+  blue1: '#4782D7',
   red: '#FF2E63',
-  holytosca: '#00ADB5',
-  holytosca30percent: '#00ADB54D',
-  blackgrey: '#828282',
+  holytosca: '#003848',
+  holytosca30percent: '#0038484D',
+  blackgrey: '#69707C',
   redalert: '#FF2E63',
   halfBaked: '#88CDD0',
   tradewind: '#58B1B5',
   elm: '#22878B',
   blockColor: '#FF2E63',
-  darkBlue: '#2C67BC',
+  darkBlue: '#4782D7',
   blueSea10: '#55C2FF',
   greenDark: '#004346',
-  babyBlue: '#7B9DCD'
+  babyBlue: '#7B9DCD',
+
+  signed_primary: '#4782D7',
+  signed_secondary: '#6295DD',
+  anon_primary: '#003848',
+  anon_secondary: '#154B5B',
+  radical_red: '#FF3466',
+  light_silver: '#D2D4DB'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color changes seem to be consistent and well thought out. However, I noticed that the `lightgrey` color is now set to `#FFFFFF`, which is the same as `white`. This could potentially lead to confusion in the future. If this was intentional and you want to keep the `lightgrey` variable for backward compatibility, consider adding a comment to clarify this.
```diff
-  lightgrey: '#FFFFFF',
+  lightgrey: '#F5F5F5', // or any other suitable light grey color
```
Also, the `black` color has been changed from `#000000` to `#0C0C0C`. If this change was intentional and you want to keep the `black` variable for backward compatibility, consider adding a comment to clarify this.
```diff
-  black: '#0C0C0C',
+  black: '#000000', // or any other suitable black color
```
Lastly, the `gray` and `gray1` colors have been significantly darkened compared to their previous values. If this was an intentional design decision, no action is needed. However, if these colors are used in text elements, ensure that they still provide sufficient contrast against their backgrounds for readability.
```diff
-  gray: '#69707C',
+  gray: '#828282', // or any other suitable gray color
-  gray1: '#9B9FA9',
+  gray1: '#C4C4C4', // or any other suitable gray color
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->